### ISSUE TITLE
Docs search should show color name results

### ISSUE THREAD

**Issue by jpolitz** [brownplt/pyret-docs#43]
![image](https://user-images.githubusercontent.com/119891/43604729-5784cd08-964b-11e8-84cd-ecca52f67108.png)

Nothing for yellow!

**Comment by blerner** [comment]
I'm very torn on this, because there are 150ish color names, few of which are common enough to warrant index entries...

**Comment by jpolitz** [comment]
Yeah, I also realized after I posted this that the natural next thing to
try, typing "color", works really well!


On August 2, 2018 at 1:53:26 PM, Ben Lerner (notifications@github.com)
wrote:

I'm very torn on this, because there are 150ish color names, few of which
are common enough to warrant index entries...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/brownplt/pyret-docs/issues/43#issuecomment-410064467>,
or mute the thread

**Comment by blerner** [comment]
Ok to close this, if `color` shows up well?
